bypass C++11 compatibility check, since it's unaware of Intel compilers (onlt knows about GCC & Clang)
author: Kenneth Hoste (HPC-UGent)
--- CMakeLists.txt.orig	2015-09-06 04:17:26.000000000 +0200
+++ CMakeLists.txt	2015-09-25 15:37:42.256218132 +0200
@@ -119,7 +119,8 @@
         set (PTHREAD_LIB "pthread")
     endif()
 else ()
-    message(FATAL_ERROR "Your C++ compiler does not support C++11.")
+    #message(FATAL_ERROR "Your C++ compiler does not support C++11.")
+    message(WARNING "Uknown C++ compiler (${CMAKE_CXX_COMPILER_ID}, going to assume that it supports C++11")
 endif ()
 
 ## TODO: Figure out how to detect this automatically
